languages/rust/exercises/concept/structs/.docs/introduction.md

```diff
@@ -0,0 +1 @@
+Structs in Rust are one of the main ways of creating a new type by composing other types. Structs come in three flavors: structs with named fields, tuple structs, and unit structs. For this concept exercise, we'll be exploring the first variant: structs with named fields. 
```

Comment:
I like how you've kept the introduction very small! That said, if I'm reading the `design.md` document correctly, this exercise will be introducing the students to structs, which will be a new form of syntax for students. Whenever new syntax is introduced, the `introduction.md` document [should explain and show the syntax](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsintroductionmd).

For an example, [see this file](https://github.com/exercism/v3/blob/master/languages/csharp/exercises/concept/enums/.docs/introduction.md), where new syntax is shown using an example.